### ISSUE TITLE
FastClick (touchstart) makes double tapping required for Capacitor camera plugin

### ISSUE THREAD

**Issue by MatthewSH** [quasarframework/quasar#11582]
**Describe the bug**
FastClick breaks on specific edge cases with capacitor camera. I've spent _way_ too much time tracking this issue down.
If you use [Capacitor camera](https://capacitorjs.com/docs/apis/camera) and you build your app for PWA then on the camera modal it requires you double-tap to hit any of the buttons, however no other elements through the entire app have this issue, and it only happens when you add it as a PWA, if you open it in Safari it works completely fine. (which makes sense based on [this in client-entry](https://github.com/quasarframework/quasar/blob/dev/app-webpack/templates/entry/client-entry.js#L62), also the vite version [here](https://github.com/quasarframework/quasar/blob/dev/app-vite/templates/entry/client-entry.js#L62)),

I tracked this down by using Safari web inspector and disabling event listeners after troubleshooting through everything else. Eventually found the culprit (technically it was an "anonymous function" within FastClick but once I added a breakpoint it linked directly to it. Disable that event listener and the modal works just fine, enable it and this bug occurs.

I'm not sure if this has anything to do with the fact that it's Stencil web components for capacitor because of their PWA elements. I don't know what the right answer is and I'm still troubleshooting and trying to find a solution.

**Platform (please complete the following information):**
Quasar Version: 2.0.4
@quasar/app Version: 3.1.0
@capacitor/core - 3.2.2
@capacitor/camera - 1.0.5

-------------

Looks like our versions are a bit out of date, the fastclick files don't look updated, but I'll do a full upgrade on everything and test again, once that is done I'll report back. Other than that, I have no idea.

**Comment by MatthewSH** [comment]
Update: upgrades our packages and it's still messing up...maybe this is a capacitor/pwa-elements/plugins issue? Want to get input here first.

Quasar: 2.3.4
@quasar/app: 3.2.5
@capacitor/core: 3.3.2 
@capacitor/camera: 1.2.1

I'll probably have to do some nasty DOM workaround since I can't remove fastclick, I can't bypass it, the `needsclicked` doesn't work, etc.

Anyone else experience this?

**Comment by jaybo** [comment]
I seem to be experiencing this issue with mapbox geocoder results within a `q-card-section`.  Only on iOS/PWA the click handler for autocompletion results fails to fire.  This link implies that `needsclick` should to be added to each result:
https://github.com/mapbox/mapbox-gl-geocoder/issues/49#issuecomment-619263907

Update:  Added `needsclick` as in the referenced link fixed my issue!

**Comment by MatthewSH** [comment]
Just want to add a reference to another issue: #13619

**Comment by rstoenescu** [comment]
Our fastclick has been removed some time ago. Forgot to close this ticket.
